CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.3.0 (Unreleased)
```

Comment:
1.2.0 is not released yet! Please add these change log to the section below.

Reply:
Done